template/web/app/buy-me-coffee/_components/InputText.tsx

```diff
@@ -0,0 +1,30 @@
+import clsx from 'clsx';
+
+export default function InputText({
```

Comment:
Site note for next PRs, start using https://ui.shadcn.com/ for general components.

Maybe we have a way to style theme in a shared way